src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java

```diff
@@ -178,16 +189,28 @@ void shouldDownloadDocumentFromDocumentManagement_FromCaseDocumentClientApi() th
             .getDocumentBinary(anyString(), anyString(), eq(documentId));
     }
 
+    @Test
+    void shouldReturnExpectedGeneratedDocument() throws Exception {
+        CaseData caseData = CaseDataBuilder.builder().atStateClaimSubmitted()
```

Comment:
Put a `//Given` comment above this line. Ideally with the conditions of the given

Example:
```suggestion
        // Given: a case in state Claim Submitted
        CaseData caseData = CaseDataBuilder.builder().atStateClaimSubmitted()
```